controllers/games-controllers.js

```diff
@@ -67,11 +69,22 @@ const patchReview = (req, res, next) => {
 		});
 };
 
+const deleteComment = (req, res, next) => {
+	const { comment_id } = req.params;
+	removeCommentById(comment_id)
+		.then((comment) => {
+      res.status(204).send({comment});
```

Comment:
Express will not send back a response body so one should not be included in the code. res.status(204).send() would be acceptable in this case. 👍